scripts/lib/CIME/XML/standard_module_setup.py

```diff
@@ -1,11 +1,9 @@
 
 # pragma pylint: disable=unused-import
 
-import logging
-import os
-import sys
-import xml.etree.ElementTree as ET
```

Comment:
We do not want anyone besides generic_xml making direct calls to ElementTree, so don't import it by default.